### PR TITLE
feat(launcher): explicitly set font-render-hinting to medium

### PR DESCRIPTION
**Pull request by cfellin1**
In response to #2410.

Snapshot of properly aligned text:

text-alignment

Looking for feedback on snapshot test which
1) is probably not in the right place within the testing suite
2) is failing for me on Linux, with the snapshot being different from the expectation, but not visibly so.

### CODE REVIEW

File: test/page.spec.js [hunk 1]
```diff
@@ -1659,4 +1659,13 @@ module.exports.addTests = function({testRunner, expect, puppeteer, DeviceDescrip
       expect(page.browser()).toBe(browser);
     });
   });
+
+  describe('Page', function() {
+    fit('should render text consistently', async function({ page, server }) {
```
Comment (JoelEinbinder) [inline test/page.spec.js]: s/fit/it

File: lib/Launcher.js [hunk 2]
```diff
@@ -45,6 +45,7 @@ const DEFAULT_ARGS = [
   '--disable-prompt-on-repost',
   '--disable-sync',
   '--disable-translate',
+  '--font-render-hinting=medium',
```
Comment (JoelEinbinder) [inline lib/Launcher.js:48]: If this is really better, we should probably set it as the default in headless chromium and not pass it in with Puppeteer. @caseq ptal.

File: test/assets/text-alignment.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    font-size: 15px;
+  }
+  div.a {
+    line-height: 0.5;
+  }
+  </style>
+</head>
+<body>
+  <div class="a">
+    <div>aaaaa..............................................................................|111</div><br>
```
Comment (JoelEinbinder) [inline test/assets/text-alignment.html:17]: We don't have consistent font rendering across operating systems, so this test is going to be tricky to make work. I am not an expert on fonts, but I imagine there is more than just the hinting that will make this test not pixel perfect.

Comment (cfellin1) [reply test/assets/text-alignment.html:17]: Yeah, that's consistent with my experiences thus far. How would you recommend testing the change?

Comment (cfellin1) [reply test/assets/text-alignment.html:17]: (assuming this is even the approach we decide on)

Comment (JoelEinbinder) [reply test/assets/text-alignment.html:17]: We could launch the page and headless and headful, and check that the screenshots are the same. Look at the headful.spec.js for examples of that.

**Comment by aslushnikov** [comment]
@cfellin1 @JoelEinbinder I looked more into this. We need to fix chromium to make headful respect the `--font-render-hinting` flag. Otherwise there's no guarantee font hinting will be the same between headful and headless.

**Comment by aslushnikov** [comment]
Closing this until upstream is fixed.

**Pull request closed without merging**